docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -285,18 +285,18 @@ Certora Prover is solving.
 Sometimes it helps to choose a selection and prioritization of solvers that
```

Comment:
I'm not sure I like what remains of the [Dealing with nonlinear arithmetic](https://certora-certora-prover-documentation--219.com.readthedocs.build/en/219/docs/user-guide/timeouts/timeouts-main.html#dealing-with-nonlinear-arithmetic) section. But we can deal with that later.